docker-compose.yml

```diff
@@ -18,7 +18,7 @@ services:
     depends_on: []
     volumes: [ ./apps/proxy:/app ]
     environment: []
-    ports: [ 80:80, 443:443 ]
+    #ports: [ 80:80, 443:443 ]
```

Comment:
The proxy is the only service we can't remove the port forwarding on as this provides the primary entry point to the stack